Tests/TestingApps/PaywallsTester/ci_scripts/ci_pre_xcodebuild.sh

```diff
@@ -1,19 +1,10 @@
 #!/bin/bash -e
 
-PAYWALLS_TESTER_API_KEY_FOR_TESTING=$REVENUECAT_XCODE_CLOUD_SIMPLE_APP_API_KEY_FOR_TESTING
-PAYWALLS_TESTER_API_KEY_FOR_DEMOS=$REVENUECAT_XCODE_CLOUD_SIMPLE_APP_API_KEY_FOR_DEMOS
 SCRIPT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
 
-if [ -z "$PAYWALLS_TESTER_API_KEY_FOR_TESTING" ]; then
-    echo "PaywallsTester API key for testing environment variable is not set."
-elif [ -z "$PAYWALLS_TESTER_API_KEY_FOR_DEMOS" ]; then
-    echo "PaywallsTester API key for demos environment variable is not set."
-else
-    echo "Replacing API keys on PaywallsTester"
+echo "Replacing API keys on PaywallsTester"
+
+file="$SCRIPT_DIR/../PaywallsTester/Keys.swift"
+sed -i.bak 's/static var api: String { "" }/static var api: String { "'"$REVENUECAT_XCODE_CLOUD_RC_APP_API_KEY"'" }/' $file
+rm $file.bak
```

Comment:
I. know this was before but you can remove this if you want by removing the `.bak` from the `sed -i.bak` command above.